src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Cms.Web/appsettings.json

```diff
@@ -29,7 +29,7 @@
       {
         "Name": "File",
         "Args": {
-          "path": "app_data/logs/orchard-log.txt",
+          "path": "App_Data/Logs/orchard-log.txt",
```

Comment:
Why you change 'logs' to 'Logs'?

Reply:
Hi hishamco,

Only so that it is consistent with other subdirectories within `App_Data`.

```
andrew@apollo ~/s/c/t/s/c/App_Data (master)> tree -L 2
.
├── Logs
│   ├── orchard-log20210909.txt
│   └── orchard-log20210910.txt
├── Sites        <-----
│   └── Default
└── tenants.json

3 directories, 3 files

```

Reply:
If is that, please fix `NLog.config` too

Reply:
do we also need to change in `.github/workflows`?

example:

https://github.com/OrchardCMS/OrchardCore/blob/5dcb873ecc438c5d014281614a715e5c6a4c2c04/.github/workflows/pr_ci.yml#L53

Reply:
Hmm, I think we need to keep the same case in all places

So changing `app_data` to `App_Data` is good and fixes the issue

Then I think it is better to keep `logs` everywhere as it is currently

So that e.g. we don't create new folders on existing case sensitive installations